docs/static/advanced-pipeline.asciidoc

```diff
@@ -107,7 +101,9 @@ Bytes served:: `bytes`
 Referrer URL:: `referrer`
 User agent:: `agent`
 
-Edit the `first-pipeline.conf` file to add the following text:
+//QUESTION: The name/value pairs above are backwards. Is this really how grok parses the data, or is this a mistake?
```

Comment:
@jsvd I only meant that shouldn't the Field Name column should be the first column in the table. Seems backwards.